swift-pipeline

```diff
@@ -220,6 +257,21 @@ if __name__ == "__main__":
             print_if_debug(f"Unable to save metadata to {metadata_filename}")
             pass
     else:
+
+        if config.matplotlib_stylesheet_multi != "default":
+            stylesheet_path = f"{config.config_directory}/{config.matplotlib_stylesheet_multi}"
+            print_if_debug(f"Applying matplotlib stylesheet at {stylesheet_path}.")
+            style.use(stylesheet_path)
+
+        script_colours = args.colours
+            
+        # Check that the output directory exists, create otherwise
+        if not os.path.isdir(args.output):
+            print(f"Output directory does not yet exist, creating it...")
```

Comment:
Same comments as above. Could this not be done before everything is ran to avoid the code duplication?